Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocMultilineTextValue.css

```diff
@@ -29,37 +41,38 @@ span.bocMultilineTextValue .content
   right: 6px;
   width: auto;
   top: 0;
-  bottom: 0.5em;
   height: auto;
 }
 
-.mozilla span.bocMultilineTextValue .content
+span.bocMultilineTextValue textarea
 {
-  right: 8px;
-  bottom: calc(0.5em + 3px);
+  width: 100%;
+  height: 100%;
+  display: inline-block;
+  min-height: var(--min-height);
 }
 
-.bocMultilineTextValue.readOnly .content
+.mozilla span.bocMultilineTextValue .content
 {
-  position: static;
+  /* Ensures that the outline of the textarea is displayed correctly. */
+  bottom: var(--mozilla-offset);
```

Comment:
Removing overflow would take care of this and no browser-specifics would be preferred.